addon/templates/components/polaris-text-field/connected.hbs

```diff
@@ -0,0 +1,21 @@
+{{#if (or left right)}}
+  <div class="Polaris-Connected">
+    {{#if left}}
+      <div class="Polaris-Connected__Item Polaris-Connected__Item--connection">
+        {{render-content left}}
```

Comment:
I think this should just be `{{component left}}` instead of `render-content`, since it doesn't make sense to render text here as the styling looks wack if you do. Also the React implementation explicitly takes a `React.ReactNode` for this property, which in our Ember port usually translates to a component. We can still accept a string though as the `component` helper will work with that too 👍